src/main/kotlin/step2/StringValidator.kt

```diff
@@ -0,0 +1,57 @@
+package step2
+
+object StringValidator {
+    /**
+     * 문자열이 유효한지 한번에 체크
+     */
+    fun validateString(string: String) {
+        string.validateFirstAndLast()
+        string.validateDuplicatedOperation()
+    }
+
+    /**
+     * 문자열의 처음과 끝이 * / 연산이면 IllegalArgumentException
+     */
+    private fun String.validateFirstAndLast() {
+        require(
+            first().isMultiplyOrDivide().not() &&
+                last().isMultiplyOrDivide().not()
+        ) { START_STRING_CAN_NOT_BE_OPERATION }
+    }
+
+    /**
+     * 연산자가 두개 이상 동시에 들어오는 경우
+     */
+    private fun String.validateDuplicatedOperation() {
+        mapIndexed { index, char ->
```

Comment:
```suggestion
        forEachIndexed { index, char ->
```
map은 반환하는 용도인데 현재와 용도가 맞지 않아보여요.

Reply:
넵 map을 쓰는게 습관이 되어있어서요! 감사합니다